src/types/coin.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  FuturesContractType,
+  FuturesSymbolOrderBookTicker,
+  OpenInterest,
+  PositionSide,
+} from './futures';
+import {
+  numberInString,
+  OrderSide,
+  SymbolFromPaginatedRequestFromId,
+} from './shared';
+
+export interface PositionRisk {
+  symbol: string;
+  positionAmt: numberInString;
+  entryPrice: numberInString;
+  markPrice: numberInString;
+  unRealizedProfit: numberInString;
+  liquidationPrice: numberInString;
+  leverage: numberInString;
+  maxQty: numberInString;
+  marginType: string;
+  isolatedMargin: numberInString;
+  isAutoAddMargin: boolean;
+  positionSide: PositionSide;
+  updateTime: number;
+}
+
+export interface CoinMOpenInterest extends OpenInterest {
+  pair: string;
+  contractType: FuturesContractType;
+}
```

Comment:
While I generally agree with extending to reduce repetition, I wonder if it makes it less readable from a usability perspective (having to dig through multiple steps of the type to see the full request schema). 

Given it's just a few lines, I'm tempted to just accept repetition for that reason instead of extending other types used in other places. Direct usage is still fine for types such as `PositionSide`, I'm just concerned for full request objects. What're your thoughts from a usage perspective? Did you have to dig through code to discover the request param types? Is one approach easier than the other?

Reply:
I agree!! I committed the change!

#a6fddb9c1c99204f31f14f6f5a716cabf459219e